### PR TITLE
3.3 explicit indexes

### PR DESCRIPTION
**Pull request by petraselmer**

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc [hunk 1]
```diff
@@ -4,31 +4,42 @@
...
-
-include::node-by-index-seek.asciidoc[leveloffset=+1]
-
-include::relationship-by-index-seek.asciidoc[leveloffset=+1]
-
-include::node-by-index-query.asciidoc[leveloffset=+1]
+The `START` clause was removed in Cypher 3.2, and the recommendation is to use `MATCH` instead (see <<query-match>>).
+However, if the use of explicit indexes is required, a series of built-in procedures, detailed below, allows these to be managed and used.
+These procedures offer the same functionality as the `START` clause.
+In addition, queries using these procedures will exhibit superior execution performance over queries using `START`.
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc]: They 'may' exhibit superior performance due to the cost planner. But it is not for certain, many simple queries performed the same between RULE and COST. I would word it:

> In addition, queries using these procedures may exhibit superior execution performance over queries using `START` due to the use of the `COST` planner and newer Cypher 3.2 compiler.

**Review by craigtaverner — APPROVED** [review]
Approved, with one suggestion.

File: cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc [hunk 2]
```diff
@@ -34,9 +34,9 @@ Using the `START` clause explicitly in a query will cause the query to fall back
...
 | `db.index.explicit.seekRelationships(indexName :: STRING?, key :: STRING?, value :: ANY?) :: (relationship :: RELATIONSHIP?)` | Get relationship from explicit index. Replaces `START r=relationship:relIndex(key = 'A')`
 | |
-| `db.index.explicit.searchRelationshipsIn(indexName :: STRING?, in :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship from explicit index, starting at the node 'in'
-| `db.index.explicit.searchRelationshipsOut(indexName :: STRING?, out :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship from explicit index, ending at the node 'out'
-| `db.index.explicit.searchRelationshipsBetween(indexName :: STRING?, in :: NODE?, out :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship from explicit index, starting at the node 'in' and ending at 'out'
+| `db.index.explicit.searchRelationshipsIn(indexName :: STRING?, in :: NODE?, query :: ANY?) :: (relationship :: RELATIONSHIP?, weight :: FLOAT?)` | Search relationship in explicit index, starting at the node 'in'
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc]: Perhaps you want to make a card to push these fixes back into the neo4j code?

Comment (petraselmer) [reply cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc]: Already done: https://github.com/neo4j/neo4j/pull/10283

**Pull request merged**